Lib/fontbakery/reporters/terminal.py

```diff
@@ -453,13 +454,18 @@ def _render_event_sync(self, print, event):
             if hasattr(msg, 'traceback'):
                 message += '\n' + '\n  ↳ '.join(msg.traceback.split('\n'))
 
-            logmsg = text_flow(message,
-                               width=76,
-                               indent=4,
-                               first_line_indent=-len(status_name)-1,
-                               left_margin=len(status_name)+1,
-                               space_padding=True)
-            print('    {} {}'.format(formatStatus(self.theme, status), logmsg))
+            formated_msg = '{} {}'.format(formatStatus(self.theme, status), message)
+            formated_msg += '\nabc *def*\n**ghi** jkl'
+            formated_msg = parse_md(formated_msg)
+            if hasattr(message, 'traceback'):
+                formated_msg += '\n' + '\n  ↳ '.join(message.traceback.split('\n'))
+            print(text_flow(formated_msg,
+                            width=76,
+                            indent=4,
+                            first_line_indent=-1-len(status_name),
+                            left_margin=6,
```

Comment:
why not `left_margin=len(status_name)+1`?

Reply:
It is an intentional negative indentation, to align the status tag aside the text content.

I add an screenshot just now to this PR description, so you can see the `INFO` positioning.
This will aling all status tags like that considering the name length.